### ISSUE TITLE
Fair Comparison with Yolo models

### ISSUE THREAD

**Issue by mrezaei92** [roboflow/rf-detr#371]
Hi,
I want to check if the performance difference primarily stems from the model architecture. If my understanding is correct, all the results you have reported come from fine-tuning your models on COCO after pretraining on Objects365. However, the Yolo models are not  pre-trained on Objects365. In such a case, the comparison does not seem to be fair.

**Comment by isaacrob-roboflow** [comment]
See table 6 in LW-DETR https://arxiv.org/pdf/2406.03459. YOLO models don't benefit nearly as much from pretraining due to architecture differences. That means unlocking the ability to leverage DINOv2 and objects365 is a major architectural advantage of DETRs, so the comparison from our perspective is fair.

**Comment by isaacrob** [comment]
Worth noting that YOLO26 does appear to pretrain on o365 :)
